content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -24,6 +24,8 @@ All **CronJob** `schedule:` times are based on the timezone of the
 If your control plane runs the kube-controller-manager in Pods or bare
 containers, the timezone set for the kube-controller-manager container determines the timezone
 that the cron job controller uses.
+
+To specify the timezone of CronJob schedule, you can get the list of timezone from [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones),  and modify timezone name, such as `UTC`, in CronJob schedule as you need.
```

Comment:
I'd replace this caution callout with a note, just below the format description, that recommends **always** setting `CRON_TZ`. How does that sound?

I'm also tempted to replace the link to https://en.wikipedia.org/wiki/Cron with link to a new page in the reference section, documenting the Golang / Kubernetes flavor of that format. We can explain that the Golang flavor is derived from Vixie cron.

Reply:
Thank you so much! It looks great!